src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestBladeoftheBlackEmpire.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MacroTools.ArtifactSystem;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+  /// <summary>
+  /// Get to Ranazjar to unlock the Aqir
```

Comment:
Not what the quest does! 

You don't have to include XML comments on quest classes anymore if you don't want to, quests are usually easy to understand anyway.